.palm/cmd_make-migrations.py

```diff
@@ -0,0 +1,17 @@
+from typing import Optional
```

Comment:
added these back, and added and `-e` flag so you don't have to stop a running container (since the ports will collide)